common/src/main/java/io/druid/audit/AuditEntry.java

```diff
@@ -91,6 +90,12 @@ public DateTime getAuditTime()
     return auditTime;
   }
 
+  @Override
+  public int compareTo(AuditEntry otherEntry)
```

Comment:
This has an inherent race condition for timers that only have resolution millisecond. Why is this needed?